Dockerfile.template

```diff
@@ -50,11 +49,11 @@ RUN dl_URL="https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-b
 	sudo ln -s /opt/gradle-* /opt/gradle && \
 	gradle --version
 
-ENV SBT_VERSION=1.6.2 \
+ENV SBT_VERSION=1.8.2 \
 	PATH=/opt/sbt/bin:$PATH
 RUN dl_URL="https://github.com/sbt/sbt/releases/download/v${SBT_VERSION}/sbt-${SBT_VERSION}.tgz" && \
 	curl -sSL --fail --retry 3 $dl_URL -o sbt.tar.gz && \
 	sudo tar -xzf sbt.tar.gz -C /opt/ && \
 	sudo ln -s /opt/sbt-* /opt/sbt && \
 	sbt about && \
-	rm -r sbt.tar.gz ~/project/project ~/project/target
```

Comment:
Instead of removing this completely, it might make more sense to use the `-f` flag. This way, if it's there, it gets deleted. If it's not, it won't fail.

This could prevent a scenario where this change was unintentional and they bring this behavior back.

Thoughts?

Reply:
i think that's a great idea